superset/db_engine_specs/ocient.py

```diff
@@ -152,67 +156,71 @@ def _find_columns_to_sanitize(cursor: Any) -> List[PlacedSanitizeFunc]:
         if cursor.description[i][1] in _sanitized_ocient_type_codes
     ]
 
-    
+
 class OcientEngineSpec(BaseEngineSpec):
-    engine = 'ocient'
+    engine = "ocient"
     engine_name = "Ocient"
-    #limit_method = LimitMethod.WRAP_SQL
+    # limit_method = LimitMethod.WRAP_SQL
     force_column_alias_quotes = True
     max_column_name_length = 30
 
     # Store mapping of superset Query id -> Ocient ID
     # These are inserted into the cache when executing the query
     # They are then removed, either upon cancellation or query completion
-    query_id_mapping: Dict[str, str]= dict()
+    query_id_mapping: Dict[str, str] = dict()
     query_id_mapping_lock = threading.Lock()
 
-    custom_errors : Dict[Pattern[str], Tuple[str, SupersetErrorType, Dict[str, Any]]] = {
+    custom_errors: Dict[Pattern[str], Tuple[str, SupersetErrorType, Dict[str, Any]]] = {
         CONNECTION_INVALID_USERNAME_REGEX: (
             __('The username "%(username)s" does not exist.'),
             SupersetErrorType.CONNECTION_INVALID_USERNAME_ERROR,
             {},
         ),
         CONNECTION_INVALID_PASSWORD_REGEX: (
-            __('The user/password combination is not valid (Incorrect password for user).'),
+            __(
```

Comment:
is `__` a function?

Reply:
or name mangling python voodoo?

Reply:
yes, __ is the frontend call for `flask_babel`

Reply:
I don't like it, but that is how all of the dialects do it